src/components/visx/TimeSeriesChart.tsx

```diff
@@ -151,7 +151,7 @@ export const TimeSeriesChart = <Datum extends {}>({
     if (defaultZoomDomain && defaultZoomDomain !== zoomDomain) {
       setZoomDomainAnimateTo(defaultZoomDomain);
     }
-  }, [defaultZoomDomain, zoomDomain]);
+  }, [defaultZoomDomain]);
```

Comment:
eslint disable so we know it's on purpose I vote

Reply:
hmm sry I've been poking around this file for a bit, trying to untangle the effect logic of this file, I thinkkkk this can basically just be written as 

```
useEffect(() => 
  if (defaultZoomDomain) {
    setZoomDomainAnimateTo(defaultZoomDomain);
  },
}, [defaultZoomDomain]) 
```

which would enable us to avoid the disable statement 🤔 . Poking around a bit more to see if there are edge cases I'm missing, but otherwise, I think this effectively just says "animate the zoom level when the caller passes in a new default" which should always be what we want (if current zoom is equivalent, the user shouldn't see anything new)